css/src/components/tabs.scss

```diff
@@ -0,0 +1,120 @@
+$tabs-padding: 0.5em 1em !default;
```

Comment:
Sorry to be a pain, but look like we'll need to rename to make room for a more standard .tabs control

Reply:
Sounds like segemented-control is our winner

Reply:
updated